src/uuid_worker.py

```diff
@@ -1067,56 +1055,25 @@ def get_file_info(self, entity_id):
                 results = [dict((curs.description[i][0].lower(), value) for i, value in enumerate(row)) for row in
                            curs.fetchall()]
 
-        # remove unneeded result columns and change the name of base_id to file_uuid
-        for item in results:
-            item['file_uuid'] = item['uuid']
-            item.pop('uuid')
-            # item.pop('descendant_uuid')
-            item.pop('ancestor_uuid')
-        return results
-
-''' 
-    def newUUID(self, parentID, entityType, userId, userEmail, submissionId=None):
-        uuid = self.uuidGen() #uuid.uuid4().hex
-        hubmap_id = self.hmidGen()
-
-        with self.lock:
-            count = 0
-            while(self.uuidExists(uuid) and count < 100):
-                uuid = self.uuidGen() #uuid.uuid4().hex
-                count = count + 1
-            if count == 100:
-                raise Exception("Unable to generate a unique uuid after 100 attempts")
-
-            count = 0;
-            while(self.doiExists(hubmap_id) and count < 100):
-                hubmap_id = self.hmidGen()
-                count = count + 1
-            if count == 100:
-                raise Exception("Unable to generate a unique hubmap id after 100 attempts")                 
-            now = time.strftime('%Y-%m-%d %H:%M:%S')
-            sql = "INSERT INTO hm_uuids (HMUUID, DOI_SUFFIX, ENTITY_TYPE, PARENT_UUID, TIME_GENERATED, USER_ID, USER_EMAIL, HUBMAP_ID)
-             VALUES (%s, %s, %s, %s, %s, %s,%s, %s)"
-            vals = (uuid, hubmap_id, entityType, parentID, now, userId, userEmail, submissionId)
-            with closing(self.hmdb.getDBConnection()) as dbConn:
-                with closing(dbConn.cursor()) as curs:
-                    curs.execute(sql, vals)
-                dbConn.commit()
-
-        disp_hubmap_id = self.__displayDoi(hubmap_id)
-        if submissionId is None:
-            rVal = {
-                "displayDoi": disp_hubmap_id,
-                "doi": hubmap_id,
-                "uuid": uuid
-                }
+        results_json = json.dumps(results)
+        if len(results_json) < self.large_response_threshold:
```

Comment:
When calculating the size, we may take the encoding into consideration. Use `len(results_json.encode('utf-8'))` for  single-character strings that require multiple bytes. However, I don't really know what exactly is stored in the db.

Reply:
I mentioned the encoding when discussing with Bill, and he didn't think we would be concerned about multibyte characters too soon.  But I thought by operating at a 9Mb threshold, we were unlikely to bump into problem.

Reply:
Code change tested locally and pushed.